_posts/en/newsletters/2022-12-14-newsletter.md

```diff
@@ -0,0 +1,242 @@
+      subsequent payment attempts.
+
+        However, the LN protocol doesn't currently have a widely
+        deployed way to determine which channel delayed an HTLC, so it's
+        less consequential in this regard to delay forwarding an HTLC
+        than it is to outright refuse to forward it.  Jager notes that
+        this is changing due to many LN implementations working on
+        supporting more detailed onion-routed error messages (see
+        [Newsletters #224][news224 fat]), so this advantage may
+        disappear some day.
+
+    Jager calls CircuitBreaker, "a simple but imperfect way to deal with
+    channel jamming and spamming".  Work continues on finding and
+    deploying a protocol-level change that will more comprehensively
+    mitigate concerns about jamming attacks, but CircuitBreaker stands
+    out as a seemingly reasonable solution that's compatible with the
+    current LN protocol and which any LND user can deploy immediately on
+    their forwarding node.  CircuitBreaker is MIT licensed and
+    conceptually simple, so it should be possible to adapt or port for
+    other LN implementations.
+
+- **Monitoring of full-RBF replacements:** developer 0xB10C
+  [posted][0xb10c rbf] to the Bitcoin-Dev mailing list that they've
+  begun providing [publicly accessible][rbf mpo] monitoring of
+  transaction replacements in the mempool of their Bitcoin Core node
+  that don't contain the BIP125 signal.  Their node allows full-RBF
+  replacement using the `mempoolfullrbf` configuration option (see
+  [Newsletter #208][news208 rbf]).
+
+    Users and services can use the website as an indicator for which
+    large mining pools might be currently confirming unsignaled
+    replacement transactions (if any are doing so).  However, we remind
+    readers that payments received in unconfirmed transactions cannot be
+    guaranteed even if miners don't currently seem to be mining
+    unsignaled replacements.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday (TODO: check for Bitcoin Core 24.0.1)
```

Comment:
The binaries are now available, and the website should be updated to reflect this release today.